src/pages/hardware/protocolReference/can.tsx

```diff
@@ -0,0 +1,404 @@
+import { ReactNode } from "react";
+import { Page } from "../../../components";
+import { HeaderSections } from "../../../components/Header/headerSectionsEnum";
+import Accordion from "../../../components/Accordion/Accordion";
+import { TableOfContentsItem } from "../../../components/TableOfContents/TableOfContents";
+import { PageTitleProps } from "../../../components/PageTitle/PageTitle";
+import { CodeSample } from "../../../components/CodeSamplesContainer";
+
+const integrationProcess: ReactNode = (
+    <div className="paragraph">
+        <p>The following process should be followed when integrating a third-party device with the GO device using our Third-Party Data CAN Protocol:</p>
+        <h2>1 - Request External Device ID</h2>
+        <p>Contact <a href="mailto:soleng@geotab.com" target="_blank" rel="noopener noreferrer">Geotab Solution Engineering team</a> to obtain an External Device ID for your third-party device. This will
+            allow us to uniquely identify your device when it connects to a GO device. Thereafter, any connection established by that type of external device will be recorded as Status Data in MyGeotab
+            under the naming format "[External device name] device connected".</p>
+        <h2>2 - Request Status Data IDs</h2>
+        <p>There is an extensively defined Status Data ID list which can be found at <a href="https://docs.google.com/spreadsheets/d/1sy7IOhWmFoo40_I-ruOJO8bVTMLXqHa11d0oJtaoIcE/edit#gid=1133172080"
+            target="_blank" rel="noopener noreferrer">MyGeotab Diagnostics</a>. Specifics regarding Status Data ID implementations can be found on the README sheet. </p>
+        <h2>3 - Implement the Third-Party CAN Protocol</h2>
+        <p>Implement the Third-Party CAN Protocol in the external device as described below. The CAN speed to be used will be 250K or 500K, and the external device should have its CAN transceiver set to
+            normal mode. The IOX CAN will auto-baud between 250K and 500K.
+        </p>
+        <h2>CAN ID</h2>
+        <p>The CAN ID is an extended frame message (29-bit) and is broken down into 4 bytes, with the most significant byte (MSB) (byte 1) containing 5 bits to make up the 29-bit ID header. A breakdown
+            of the CAN ID is shown below:</p>
+        <div className="table-container">
```

Comment:
Not quite sure if the .table-container and/or table styling will be updated in the future, but it currently appears to have a lot of dead space and takes up the 100% of the container so on larger screens the table extends too much instead of fitting the contents